src/main/java/org/core/client/UserClient.java

```diff
@@ -53,4 +54,24 @@ public int postUsers(User user) {
             throw new RuntimeException(e);
         }
     }
+
+    public int putUser(UserToUpdate user) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            HttpResponse response = Client.doPut(POST_USERS_ENDPOINT, objectMapper.writeValueAsString(user));
+            return response.getStatusLine().getStatusCode();
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public User createAvailableUser(User user) {
+        UserClient userClient = new UserClient();
```

Comment:
no need to create it

try this ex.:
```
public User createAvailableUser(String zipcode) {
        User user = new User(RandomUtils.nextInt(0, 120),
                RandomStringUtils.randomAlphabetic(10),
                Gender.FEMALE, zipcode);
        int statusCode = postUsers(user);
        if(statusCode == 201) {
            return user;
        } else {
            throw new RuntimeException("Failed to create available user. Check POST /users method.");
        }
    }
```
the idea is to make the tests work even if you have an empty database.

btw, make postUsers(); method singular. As far as I remember it is impossible to post list of users